agent.go

```diff
@@ -122,15 +124,15 @@ func (p *process) closePostExitFDs() {
 	}
 }
 
-func (c *container) setProcess(pid int, process *process) {
+func (c *container) setProcess(execID string, process *process) {
 	c.Lock()
-	c.processes[pid] = process
+	c.processes[execID] = process
```

Comment:
StartContainer() calls a.runProcess(req.ContainerId, "", nil).
Will the control flow go to here?

Reply:
What do you mean by control flow ? `runProcess()` is a global function designed to handle both `StartContainer()` and `ExecProcess()`

Reply:
the execID is "" in this case.
Should container init process save in the c.processes? I'm Ok with any answers.
Is c.processes[""] = process.  (I never try it before ... let me check it in the playground)

Reply:
Sorry, ignore this...